src/components/icon/icon.jsx

```diff
@@ -80,8 +90,10 @@ Icon.defaultProps = {
   fill: 'none',
   id: undefined,
   onClick: () => {},
+  onEnter: () => {},
   role: 'img',
   size: 'medium',
   stroke: 'none',
   title: undefined,
+  active: true,
```

Comment:
This should probably be false, so multiple icons on the same page aren't all active.